### PR TITLE
Make add mission button and queue independent

### LINKED ISSUE #1173: Move "Add mission"-button
**Describe the improvement you would like to see**
Move the "Add mission"-button and remove the "Create mission"-button from missionQueue section. 

We mainly rely on inspection plan and not echo missions anymore. The echo mission related "Add mission"-button should therefor no longer be in the center of the frontpage. Suggested new placement in image below

Current placement:
image

Suggested new placement:
image

**How will this change existing functionality?**
It well be more intuitive to use the inspection plan instead of echo missions

**How will this improvement affect the current Threat Model?**
N/A

### PR DESCRIPTION
**Pull request by mrica-equinor**
Closes #1173

### CODE REVIEW

File: frontend/src/components/Pages/FrontPage/MissionOverview/ScheduleMissionDialog/ScheduleMissionDialog.tsx [hunk 1]
```diff
@@ -38,6 +35,8 @@ export const ScheduleMissionDialog = ({ setLoadingMissionSet }: IProps): JSX.Ele
     const [echoMissions, setEchoMissions] = useState<Map<string, EchoMissionDefinition>>(
         new Map<string, EchoMissionDefinition>()
     )
+    const { setLoadingMissionSet } = useMissionsContext()
```
Comment (Eddasol) [inline frontend/src/components/Pages/FrontPage/MissionOverview/ScheduleMissionDialog/ScheduleMissionDialog.tsx:38]: Cluster contexts together at the top of the function

**Comment by Eddasol** [comment]
Suggest removing the buttons from mission queue view and changing the "add mission"-button text to something like "Add predefined echo mission"

**Review by andchiind — APPROVED** [review]
LGTM

**Review by eivindsjovold — APPROVED** [review]
Did not test locally™️ but LGTM

**Pull request merged**